cfn-resources/teams/test/README.md

```diff
@@ -35,4 +35,25 @@ The Atlas organization must be manually provided.
 
 ## Important Links
 - [API Documentation](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Teams)
-- [Resource Usage Documentation](https://www.mongodb.com/docs/atlas/access/manage-teams-in-orgs/)
\ No newline at end of file
+- [Resource Usage Documentation](https://www.mongodb.com/docs/atlas/access/manage-teams-in-orgs/)
+
+## Unit Testing Locally
+
+The local tests are integrated with the AWS `sam local` and `cfn invoke` tooling features:
+
+```
+sam local start-lambda --skip-pull-image
+```
+then in another shell:
+```bash
+repo_root=$(git rev-parse --show-toplevel)
+source <(${repo_root}/quickstart-mongodb-atlas/scripts/export-mongocli-config.py)
+cd ${repo_root}/cfn-resources/teams
+./test/teams.create-sample-cfn-request.sh {TEAM_NAME},{ ORGANIZATION_USER_EMAIL} > test.request.json 
+echo "Sample request:"
+cat test.request.json
+cfn invoke CREATE test.request.json 
+cfn invoke DELETE test.request.json 
```

Comment:
if i dont remember wrong the cfn test was the one used for Unit Testing (all the contract testings), the cfn invoke was used to actually execute the resource

Reply:
Yes, there shouldn't be any changes to the logic @Adelmar92